tests/property/test_group.py

```diff
@@ -143,8 +143,8 @@ def test_simple_powers(self):
     @given(elements_mod_q())
     def test_in_bounds_q(self, q: ElementModQ):
         self.assertTrue(q.is_in_bounds())
-        too_big = q + get_small_prime()
-        too_small = q - get_small_prime()
+        too_big = int(q) + get_small_prime()
+        too_small = int(q) - get_small_prime()
```

Comment:
Why does this have to be cast to an int? 

My thought would be 

Assert that:
- `ElementModQ + small prime int => too big ElementModQ` 
     - If ElementModQ is too big after addition, Overflow error should be tossed

- `int + small prime int = > too big int`
    - If ElementModQ is constructed with too big int, Overflow error should be tossed
    - Also validate the is_in_bounds works